### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Drigax**
update current project capabilities

### CODE REVIEW

**Comment by Drigax** [comment]
We should also update this to ensure the Linux build instructions are accurate.

**Comment by Drigax** [comment]
Linux instructions seem accurate from what I can tell, I was able to get started from fresh install by following the current guide. I'm not sure what we may be missing...

File: README.md [hunk 1]
```diff
@@ -37,15 +37,13 @@ The following major features are partially implemented but not yet supported. No
...
-- Building and running the libraries and demo apps for the following platforms: 
-Android.
 - Mixed reality powered by OpenXR on the following platforms: Win32, UWP.
+- User Input.
 
 The following major features are not yet supported or implemented, even as previews, 
 but are expected to be supported in the future. Note that this list is not exhaustive.
 
 - Debugging JavaScript with V8.
-- User input.
```
Comment (bghgary) [inline README.md:48]: We still don't have this yet. Don't remove it.

File: README.md [hunk 2]
```diff
@@ -350,8 +348,13 @@ Then targeting a Ninja make file:
 cmake -GNinja -DJSCORE_LIBRARY=/usr/lib/x86_64-linux-gnu/libjavascriptcoregtk-4.0.so ..
 ```
 
-Ninja is not mandatory and make can be used instead.
-And finaly, run a build:
+Ninja is not mandatory and make can be used instead:
```
Comment (bghgary) [inline README.md:351]: If `ninja-build` is a dependency, we should just tell people to use ninja to build. Having a fork at each step seems confusing to me.

Comment (Drigax) [reply README.md:351]: makes sense, I'll just remove the additional instruction.

**Review by bghgary — APPROVED** [review]

**Pull request merged**